src/abilities/Headless.js

```diff
@@ -299,13 +299,15 @@ export default (G) => {
 				}); //remove creatures
 				ability.end();
 
-				//Movement
+				// Movement
 				arrayUtils.filterCreature(path, false, false);
 				let destination = null;
 				let destinationTarget = null;
 				if (target.size === 1) {
-					// Small creature, pull target towards self
-					destinationTarget = path.first();
+					/* Small creature, pull target towards self landing it in the hex directly
+					in front of the Headless. */
+					const hexInFrontOfHeadless = path[0];
```

Comment:
The MR also fixes a small bug completely preventing the Headless Whip Move ability from working. There are still plenty of other bugs with it, but this at least gets it operational ... sometimes.